src/seedu/addressbook/commands/Command.java

```diff
@@ -54,6 +54,10 @@ public void setData(AddressBook addressBook, List<? extends ReadOnlyPerson> rele
         this.relevantPersons = relevantPersons;
     }
 
+    public boolean isMutating() {
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves =).